apps/parsley/docs/06-Downloading-Logs.md

```diff
@@ -7,67 +7,68 @@ Note that there is a 1 minute timeout on log downloads. If you find that your do
 The methods listed below are ordered from most to least recommended. We recommend going through these methods sequentially if you are running into issues fetching your logs.
 
 ## Via Evergreen CLI
+
 You can use the following commands with the `evergreen` client to fetch task and test logs.
 
-#### Task Logs
+### Task Logs
+
 ```properties
 evergreen task build TaskLogs --task_id <task_id> --execution <execution> --type <task_log_type>  --o output.txt
 ```
 
 See other options using the following command:
+
 ```properties
 evergreen task build TaskLogs --help
 ```
 
-#### Test Logs
+### Test Logs
+
 ```properties
 evergreen task build TestLogs --task_id <task_id> --execution <execution> --log_path <test_log_path> --o output.txt
 ```
 
 See other options using the following command:
+
 ```properties
 evergreen task build TestLogs --help
 ```
 
-To find the correct value for `<test_log_path>`, we recommend viewing `https://evergreen.mongodb.com/rest/v2/tasks/<task_id>/tests` ([docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tests/paths/~1tasks~1%7Btask_id%7D~1tests/get)) in your browser and using Ctrl + F to locate your test. 
+To find the correct value for `<test_log_path>`, we recommend viewing `https://evergreen.corp.mongodb.com/rest/v2/tasks/<task_id>/tests` ([docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tests/paths/~1tasks~1%7Btask_id%7D~1tests/get)) in your browser and using Ctrl + F to locate your test.
 
 Please note that this endpoint will only show the first 100 tests by default; append the `?limit=<num>` query parameter to show more tests. The test log path can then be grabbed from the `logs.url` field which should contain some value `test_name=<this_is_the_test_log_path>`.
 
 ## Via Spruce
 
 Right click the "Raw" button and select the "Save Link As..." option. This will save the logs as a `.txt` file.
 
-#### Task Logs
+### Task Logs
+
 ![task logs button](./images/task_logs_button.png)
 
-#### Test Logs
+### Test Logs
+
 ![test logs button](./images/test_logs_button.png)
 
-## Via Terminal
+## Via Rest API
```

Comment:
I rebranded this section to with rest api to be more generic. The linked authentication docs shows usage with a tool like `curl`. I figured it would be troublesome to continue to support both just in case for the future.